egs2/lrs2/lipreading1/local/feature_extract/video_processing.py

```diff
@@ -0,0 +1,212 @@
+import skvideo.io
+import skimage.transform
```

Comment:
Do we already have this package as the requirement of espnet? If not, maybe do the check somewhere before (e.g., data.sh or path.sh)

Reply:
No, we don't have this. But I add a check in `data.sh` to remind users to install it manually. Do you think it is better to ad d it to the reqirements or we just remind user install it by themself?

Reply:
Oh, sorry, I did not see the check. I think the current version is already in a good shape. Thanks